.github/workflows/qa.yml

```diff
@@ -9,21 +9,24 @@ on:
 jobs:
   phpunit:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ 7.2, 7.4, 8.0 ]
```

Comment:
I'm not sure that I see the value of running phpunit on multiple php versions. Do you have a use case where this could make a difference?